src/app/shell/personal-cabinet/provider/provider-workshops/provider-workshops.component.ts

```
@@ -31,9 +32,14 @@ import { ProviderComponent } from '../provider.component';
 export class ProviderWorkshopsComponent extends ProviderComponent implements OnInit, OnDestroy {
   @Select(ProviderState.providerWorkshops)
   public workshops$: Observable<SearchResponse<WorkshopProviderViewCard[]>>;
+  @Select(ProviderState.hasDraftData)
+  public hasDraftData$: Observable<boolean>;
+  @Select(ProviderState.getTimeToLiveDraft)
+  public draftLiveTime$: Observable<string>;
 
   public readonly constants: typeof Constants = Constants;
   public readonly ModeConstants = ModeConstants;
+  public isLoaded: Boolean = false;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)